### PR TITLE
Move emacs-ansible

### LINKED ISSUE #9058: Change repo for emacs-ansible
I've set up https://gitlab.com/emacs-ansible/emacs-ansible to continue development on the package. The original author, @k1LoW, has said a [fork is welcome](https://github.com/k1LoW/emacs-ansible/issues/48#issuecomment-1938020758) and has reacted positively when [I pointed out my fork](https://github.com/k1LoW/emacs-ansible/issues/48#issuecomment-2153858789).

### PR DESCRIPTION
**Pull request by hexmode**
Fixes #9058

### CODE REVIEW

**Comment by riscy** [comment]
Thanks a ton for taking this up.  The discussion you linked looks sufficient.

I recommend when we make this change, we also update the recipe to:
```elisp
(ansible
 :fetcher gitlab
 :repo "emacs-ansible/emacs-ansible"
 :files (:defaults "snippets" "dict"))
```
(`*.el` is a risky pattern to begin with and this new recipe is exactly equivalent; lmk what you think).


-----------

In the meantime, here is some maintenance that the MELPA checklist catches --

`ansible.el` with byte-compile using Emacs 29.3:
```
ansible.el:81:2: Warning: custom-declare-variable `ansible-vault-password' docstring wider than 80 characters
ansible.el:110:2: Warning: custom-declare-face `ansible-section-face' docstring wider than 80 characters
In ansible-vault-prompt-for-password:
ansible.el:273:2: Warning: docstring wider than 80 characters
In ansible-maybe-unload-snippets:
ansible.el:298:63: Warning: reference to free variable `ansible'
ansible.el:299:53: Warning: reference to free variable `yas-snippet-dirs'
ansible.el:299:11: Warning: assignment to free variable `yas-snippet-dirs'
ansible.el:303:20: Warning: global/dynamic var `ansible' lacks a prefix
In ansible:
ansible.el:316:25: Warning: reference to free variable `yas-snippet-dirs'
ansible.el:316:25: Warning: assignment to free variable `yas-snippet-dirs'
In ansible-find-root-path:
ansible.el:333:11: Warning: reference to free variable `with'
ansible.el:333:16: Warning: reference to free variable `count'
ansible.el:333:22: Warning: reference to free variable `='
ansible.el:334:11: Warning: reference to free variable `until'
ansible.el:336:11: Warning: reference to free variable `if'
ansible.el:337:11: Warning: reference to free variable `do'
ansible.el:339:11: Warning: reference to free variable `else'
ansible.el:343:11: Warning: reference to free variable `finally'
ansible.el:343:19: Warning: reference to free variable `return'
In ansible-vault:
ansible.el:363:2: Warning: docstring has wrong usage of unescaped single quotes (use \= or different quoting)
In ansible-vault-string:
ansible.el:409:2: Warning: docstring wider than 80 characters
ansible.el:409:2: Warning: docstring has wrong usage of unescaped single quotes (use \= or different quoting)
In ansible-vault-region:
ansible.el:438:2: Warning: docstring wider than 80 characters
In end of data:
ansible.el:340:15: Warning: the function `incf' is not known to be defined.
ansible.el:337:15: Warning: the function `return' is not known to be defined.
ansible.el:333:6: Warning: the function `loop' is not known to be defined.
ansible.el:317:12: Warning: the function `yas-load-directory' is not known to be defined.
ansible.el:300:6: Warning: the function `yas-reload-all' is not known to be defined.
```

`ansible.el` with package-lint 20240507.1722:
```
8 issues found:
61:16: warning: Replace deprecated `cl' with `cl-lib'.  The `cl-libify' package can help with this.
62:14: error: You should depend on (emacs "24.3") or the cl-lib package if you need `cl-lib'.
285:3: error: You should depend on (emacs "25.1") if you need `font-lock-flush'.
291:3: error: You should depend on (emacs "25.1") if you need `font-lock-flush'.
298:3: error: You should depend on (emacs "25.1") or the seq package if you need `seq-count'.
384:30: error: You should depend on (emacs "24.3") or the cl-lib package if you need `cl-loop'.
398:21: error: You should depend on (emacs "24.4") or the compat package if you need `string-trim-right'.
485:13: warning: Closing parens should not be wrapped onto new lines.
```

File: recipes/ansible [hunk 1]
```diff
@@ -1,4 +1,4 @@
 (ansible
- :fetcher github
- :repo "k1LoW/emacs-ansible"
+ :fetcher gitlab
+ :repo "emacs-ansible/emacs-ansible"
  :files ("*.el" "snippets" "dict"))
```
Comment (riscy) [inline recipes/ansible:4]: Friendly ping with the recommended recipe change -
```suggestion
 :files (:defaults "snippets" "dict"))
```

Comment (hexmode) [reply recipes/ansible]: Missed this last week.  Thanks!

**Comment by riscy** [comment]
Thank you!

**Pull request merged**